vendor/github.com/prometheus/common/config/http_config.go

```diff
@@ -323,6 +330,15 @@ func (c *HTTPClientConfig) SetDirectory(dir string) {
 	c.BearerTokenFile = JoinDir(dir, c.BearerTokenFile)
 }
 
+// atLeastTwoTrue returns true if at least any of the two of three given bools
+// are true.
+func atLeastTwoTrue(a, b, c bool) bool {
```

Comment:
This is bit too shallow function. Perhaps more useful would be to have helper like `notSet(options ...string) bool` ? or even `notSet(flags ...any) bool` that supports strings and pointers and panics otherwise

Reply:
(I know it's vendored code)

Reply:
Done in https://github.com/prometheus/common/pull/572